blog/2022/02/10years.md

```diff
@@ -456,3 +456,9 @@ I first heard about Julia in Spring 2015 from an email from my PhD advisor. The
 This first experience set the trajectory for my entire PhD, and we quickly knew that I would be doing the rest of my research projects using Julia. That summer, I predominantly worked on [MatrixNetworks.jl](https://github.com/JuliaGraphs/MatrixNetworks.jl), which included most of the graph algorithms that I and the rest of our research team would later need. It was fascinating and thrilling at the same time. It was fascinating to create something written with such simple and easy to understand code, yet have the performance of C or C++. To realize that you have the power to drill deeper, go into Julia internals, and squeeze every ounce of performance out of your code was a thrill ride. It was like, finally, there’s a language that allows you to express any algorithmic idea, without the language posing barriers.
 
 At a certain point (probably around JuliaCon 2016), I realized that Julia had become more than just a programming language to me. It gave me a community where I met some of my most favorite humans in this world and a community I learn from every day. I am so grateful that Julia exists today!
+
+## [James Fairbanks](https://www.github.com/jpfairbanks)
```

Comment:
We had to change the formatting for the section headers to avoid breaking the toc. See above.

Reply:
fixed